docs/source/integrations/databricks_development_workflow.md

```diff
@@ -0,0 +1,220 @@
+```
+
+3. Loading the extension allows you to use the `%reload_kedro` line magic to load your Kedro project. The `--env=databricks` option will make Kedro use the `databricks` environment. The `databricks` environment is defined in the PySpark Iris starter and directs Kedro to use the configuration in the `conf/databricks` directory. Add the following code to the third new cell to load your Kedro project:
+
+```ipython
+%reload_kedro /Workspace/Repos/<databricks_username>/iris-databricks --env=databricks
+```
+
+4. Loading your Kedro project with the `%reload_kedro` line magic will define four global variables in your notebook: `context`, `session`, `catalog` and `pipelines`. You will use the `session` variable to run your project. Add the following code to the fourth new cell to run your Kedro project:
+
+```ipython
+session.run()
+```
+
+After completing these steps, your notebook should match the following image:
+
+![Databricks completed notebook](../meta/images/databricks_finished_notebook.png)
+
+Run the completed notebook using the `Run All` bottom in the top right of the UI:
+
+![Databricks notebook run all](../meta/images/databricks_run_all.png)
+
+On your first run, you will be prompted to consent to analytics, type `y` or `N` in the field that appears and press `Enter`:
+
+![Databricks notebook telemetry consent](../meta/images/databricks_telemetry_consent.png)
+
+You should see logging output while the cell is running. After execution finishes, you should see output similar to the following:
+
+```bash
+...
+[08/09/22 11:23:30] INFO     Model has accuracy of 0.960 on test data.                                        nodes.py:74
+                    INFO     Saving data to 'metrics' (MetricsDataSet)...                             data_catalog.py:382
+                    INFO     Completed 3 out of 3 tasks                                           sequential_runner.py:85
+                    INFO     Pipeline execution completed successfully.                                      runner.py:89
+```
+
+## Modify your project and test the changes
+
+Now that your project has run successfully once, you can make changes using the convenience and power of your local development environment. In this section, you will change your project to use a modified classifier to create predictions on the iris data.
+
+### Modify the training / test split ratio
+
+The PySpark Iris starter uses a default 80-20 ratio of training data to test data when training the classifier. In this section, you will change this ratio to 70-30 by editing your project in your local environment, then run the modified project on Databricks to observe the different result.
+
+Open the file `<project_root>/conf/base/parameters.yml` in your local environment. Edit the line `train_fraction: 0.8` to `train_fraction: 0.7` and save your changes. Look in the terminal where `dbx sync` is running, you should see it automatically sync your changes with your Databricks repo:
```

Comment:
Yes, see line 65:

```The next step is to use dbx to sync your project to your repo. Open a new terminal instance, activate your conda environment, navigate to your project directory and start `dbx sync`:```

Maybe this needs to be put in bold, so that the reader won't miss it.